multiplex_immuno_processing/generate_alignment_parameters_for_single_position.py

```diff
@@ -0,0 +1,262 @@
+import argparse
```

Comment:
should take an argument to run on a single possition. shoudl not be its own file.

Reply:
resolved in new set of commits